gateway-service/src/main/resources/gateway-log-messages.yml

```diff
@@ -266,3 +266,17 @@ messages:
       reason: "An error occurred in the SAF Auth Service. Review the reason in the error message."
       action: "Supply a valid user and application name, and check that corresponding permissions have been set up."
 
+    # IDT messages (150 - 159)
+    - key: org.zowe.apiml.security.idt.failed
+      number: ZWEAG150
+      type: ERROR
+      text: "SAF IDT generation failed. Reason: %s"
+      reason: "An error occurred in the SAF Verify Service. Review the reason in the error message."
```

Comment:
`An error occurred in the SAF Verify Service.` -> `SAF Verify Service` sounds a little weird, is this our standard to refer to SAF? Maybe something like `An error occurred during SAF verification`?

Reply:
@janan07 Any opinion on the wording?